ruby-challenges.rb

```diff
@@ -30,3 +89,6 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+
+#I have no idea on this one!!
```

Comment:
We would have to add in new methods called pedal_faster and brake. Those methods would update the @current_speed variable. So, for pedal_faster, we can do something like:
def pedal_faster number
  @current_speed += number
end

Now when we call on pedal_faster and give it a number as an argument, the variable current_speed will be updated to whatever the current_speed was plus the number. You can follow the same concept for brake, but you just need to find a way to prevent it from going negative!